apps/opik-backend/src/test/java/com/comet/opik/infrastructure/auth/RemoteAuthTestServer.java

```diff
@@ -0,0 +1,65 @@
+package com.comet.opik.infrastructure.auth;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.apache.hc.core5.http.HttpStatus;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.InetSocketAddress;
+import java.net.ServerSocket;
+
+@NoArgsConstructor
+class RemoteAuthTestServer {
```

Comment:
Technically I'm not against writing tests, but why was it important to add these tests right now?  what are we protecting here?